sql/dolphinscheduler-postgre.sql

```diff
@@ -566,8 +566,10 @@ CREATE TABLE t_ds_task_instance (
   retry_interval int DEFAULT NULL ,
   max_retry_times int DEFAULT NULL ,
   task_instance_priority int DEFAULT NULL ,
-   worker_group varchar(64),
+  worker_group varchar(64),
```

Comment:
You can add the sql updater in this directory.
![image](https://user-images.githubusercontent.com/29545877/89790938-3cfae480-db55-11ea-8543-17e7b9d2ce82.png)